pkg/controller/logstorage/linseed/linseed_controller_test.go

```diff
@@ -269,6 +269,21 @@ var _ = Describe("LogStorage Linseed controller", func() {
 			tenant.Name = "default"
 			tenant.Namespace = tenantNS
 			tenant.Spec.ID = "test-tenant-id"
+			tenant.Spec.Indices = []operatorv1.Index{
```

Comment:
Is this needed? I would have expected we leave this test as is, but then add a new test which checks when non-standard names are given.

Reply:
I was thinking that we will have validation to check that indices are always set. And that it will be a validation failure if they are not. Since this is not expected to be used for Calico or Calico Enterprise users then I was thinking it would be reasonable to not have any defaults and require the fields be set, it will force us to provide the configuration rather than possibly using some defaults when it was not expected.

Reply:
Sounds good to me